### PR TITLE
update changelog entry for in-memory rechunking

### PR DESCRIPTION
**Pull request by fjetter**
Rendered version

image

### CODE REVIEW

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -17,6 +17,36 @@ Zero-copy P2P Array Rechunking
...
+.. code:: python
+
+  shape = (30_000, 6_000, 150) # 201.17 GiB
+  input_chunks = (60, -1, -1) # 411.99 MiB
+  output_chunks = (-1, 6, -1) # 205.99 MiB
+
+  arr = da.random.random(size, chunks=input_chunks)
+  with dask.config.set({
+      "array.rechunk.method": "p2p",
+      "distributed.p2p.disk": "disk",
```
Comment (jrbourbeau) [inline docs/source/changelog.rst:32]: I think this is supposed to be a bool

```suggestion
      "distributed.p2p.disk": True,
```

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -17,6 +17,36 @@ Zero-copy P2P Array Rechunking
 Users should see significant performance improvements when using in-memory P2P array rechunking.
 This is due to no longer copying underlying data buffers.
 
+Below shows a simple where we compare the different rechunk methods, tasks, p2p
+and p2p without disk.
```
Comment (jrbourbeau) [inline docs/source/changelog.rst:21]:
```suggestion
Below shows a simple example where we compare performance of different rechunking methods.
```

**Review by jrbourbeau — COMMENTED** [review]
Thanks @fjetter -- just a couple of minor comments

**Review by phofl — APPROVED** [review]

**Comment by phofl** [comment]
thx @fjetter

**Pull request merged**